### DISCUSSION TITLE
I want to colour the youtube logo, but without affecting the "play" and the text "Youtube".

### DISCUSSION THREAD

**Discussion by Pinovski** [stylus/stylus#2636] [Q&A] [answered]
I am using the youtube theme called "Replace red". 
`.style-scope ytd-topbar-logo-renderer path.style-scope.yt-icon{
    
    fill: white;

}
    `
I want to change the youtube logo, that box with rounded edges, I want it for example in blue colour, but the play symbol in white colour, and the youtube text (in the logo) in white colour.

How can I do it? it colours everything in one colour.

Thanks and best regards.

**Comment by vendethiel** [comment, marked as answer]
Wrong stylus. This is the stylus programming language repo.
